### PR TITLE
Logic for blocked(403 permission) all view by default.

### PR DESCRIPTION
**Pull request by ntodosiychuk**
in urls.py

import view_acl
view_acl.autodiscover()

### CODE REVIEW

File: view_acl/__init__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+def autodiscover():
+    """
+    Block all view defined in urls.py by default.
+    Return 403 status, permission denied.
+    Skipped '/admin' and etc
+    """
+    url_patterns = get_caller_name()
+    resolver = urlresolvers.get_resolver(url_patterns)
+    for url_pattern in resolver.url_patterns:
+        url_pattern._callback = permission_denied
```
Comment (pzdeb) [inline view_acl/__init__.py:15]: we don't have to force permission-denied for all url_patterns that we have. It's too risky.
Instead we need to have auth backend that authorizes user to have permissions for particular view. That backend module can be only responsible to permission_denied returning.

File: view_acl/__init__.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from django.core import urlresolvers
+from django.views.defaults import permission_denied
+
+from .utils import get_caller_name
+
```
Comment (pzdeb) [inline view_acl/__init__.py:5]: one more whiteline please after imports, here and in another module

File: view_acl/utils.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import inspect
+
+def get_caller_name(skip=2):
```
Comment (pzdeb) [inline view_acl/utils.py:3]: Does it get package hierarchy for the view? We have to recognize packages in order to distinguish views for different applications

File: view_acl/__init__.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+def autodiscover():
+    """
+    Block all view defined in urls.py by default.
+    Return 403 status, permission denied.
+    Skipped '/admin' and etc
+    """
+    url_patterns = get_caller_name()
```
Comment (pzdeb) [inline view_acl/__init__.py:12]: If we have name parameter specified to urlpatterns for the views we won't need to call get_caller_name(). The name will be used instead

Comment (ntodosiychuk) [inline view_acl/__init__.py:12]: This function get url file, like 'mysite.url'. The file from which function was called.

**Pull request closed without merging**